src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-policies/specify-when-alerts-create-incidents.mdx

```diff
@@ -44,7 +44,7 @@ Choose from two threshold types when a policy condition is met:
 * Critical (red)
 * Warning (yellow)
 
-When a policy condition violates a threshold, an incident is opened and an issue record can be created with detailed information to help you respond efficiently. The issue record may also include any other incidents occurring after the opening incident.
+When a policy condition breaks a threshold, an incident is opened and an issue record can be created with detailed information to help you respond efficiently. The issue record may also include any other incidents occurring after the opening incident.
```

Comment:
I like to use the term "breaches" instead of "breaks." The threshold is not broken, it is breached. "Breached" has a closer meaning to what we're trying to communicate.

Reply:
Thanks! I've replaced it.